pkg/controller/volume/pvcprotection/pvc_protection_controller.go

```diff
@@ -209,11 +291,11 @@ func (c *Controller) removeFinalizer(ctx context.Context, pvc *v1.PersistentVolu
 		logger.Error(err, "Error removing protection finalizer from PVC", "PVC", klog.KObj(pvc))
 		return err
 	}
-	logger.V(3).Info("Removed protection finalizer from PVC", "PVC", klog.KObj(pvc))
+	logger.Info("Removed protection finalizer from PVC", "PVC", klog.KObj(pvc))
```

Comment:
Any reason to drop the verbosity here? This might result in more log spam

Reply:
Nice catch! I probably accidentally removed this before.